src/main/java/org/mockito/internal/debugging/MockitoDebuggerImpl.java

```diff
@@ -17,6 +17,9 @@
 
     private final UnusedStubsFinder unusedStubsFinder = new UnusedStubsFinder();
 
+    /**
+     * TODO: when MockitoDebugger is deleted, delete this implementation, too
```

Comment:
Why not to mark it with `@Override` to force a deletion? AFAIR Mockito does not support Java 5 any longer.